test/functional/api/cas/init_config.py

```diff
@@ -57,6 +58,50 @@ def create_default_init_config(cls):
         cas_version = casadm_parser.get_casadm_version()
         fs_utils.write_file(opencas_conf_path, f"version={cas_version.base}")
 
+    @classmethod
+    def load_config_file(cls):
```

Comment:
Considering comment for test_startup_init_config.py lines 174+, this method might be unnecessary